app/controllers/conversations_controller.rb

```diff
@@ -80,7 +80,11 @@ def show_errors
   end
 
   def set_emoji
-    EmojiSetter.call(@conversation)
+    begin
+      EmojiSetter.call(@conversation)
+    ensure
+      add_users_to_conversation
+    end
```

Comment:
I think you could write it like this: 
```suggestion
    EmojiSetter.call(@conversation)
  ensure
    add_users_to_conversation
```

Reply:
Thanks Alvaro!